go-controller/pkg/ovn/pods_test.go

```diff
@@ -0,0 +1,286 @@
+	return &v1.Namespace{
+		Status: v1.NamespaceStatus{
+			Phase: v1.NamespaceActive,
+		},
+		ObjectMeta: newObjectMeta(name, name),
+	}
+}
+
+func portName(namespace, name string) string {
+	return namespace + "_" + name
+}
+
+var _ = Describe("OVN Pod Operations", func() {
+	var app *cli.App
+
+	BeforeEach(func() {
+		// Restore global default values before each testcase
+		config.RestoreDefaultConfig()
+
+		app = cli.NewApp()
+		app.Name = "test"
+		app.Flags = config.Flags
+	})
+
+	Context("on startup", func() {
+		const (
+			nodeName   string = "node1"
+			nodeSubnet string = "10.128.1.0/24"
+			nodeMgtIP  string = "10.128.1.2"
+			nodeGWIP   string = "10.128.1.1"
+			nodeGWCIDR string = nodeGWIP + "/24"
+			podName    string = "myPod"
+			podIP      string = "10.128.1.4"
+			podMAC     string = "11:22:33:44:55:66"
+			namespace  string = "namespace"
+		)
+
+		It("reconciles an existing pod", func() {
+			app.Action = func(ctx *cli.Context) error {
+				portName := portName(namespace, podName)
+
+				fexec := ovntest.NewFakeExec()
+				fexec.AddFakeCmd(&ovntest.ExpectedCmd{
+					Cmd:    "ovn-nbctl --timeout=15 --data=bare --no-heading --columns=name find logical_switch_port external_ids:pod=true",
+					Output: portName + "\n",
+				})
+
+				// node setup
+				fexec.AddFakeCmd(&ovntest.ExpectedCmd{
+					Cmd:    "ovn-nbctl --timeout=15 get logical_switch " + nodeName + " other-config",
+					Output: `{exclude_ips="10.128.1.2", subnet="` + nodeSubnet + `"}`,
+				})
+				fexec.AddFakeCmd(&ovntest.ExpectedCmd{
+					Cmd:    "ovn-nbctl --timeout=15 get logical_switch " + nodeName + " other-config:subnet",
+					Output: fmt.Sprintf("%q", nodeSubnet),
+				})
+				fexec.AddFakeCmdsNoOutputNoError([]string{
+					"ovn-nbctl --timeout=15 --may-exist acl-add " + nodeName + " to-lport 1001 ip4.src==" + nodeMgtIP + " allow-related",
+				})
+				// pod setup
+				fexec.AddFakeCmdsNoOutputNoError([]string{
+					"ovn-nbctl --timeout=15 --wait=sb -- --may-exist lsp-add " + nodeName + " " + portName + " -- lsp-set-addresses " + portName + " dynamic -- set logical_switch_port " + portName + " external-ids:namespace=" + namespace + " external-ids:logical_switch=" + nodeName + " external-ids:pod=true",
+				})
+				fexec.AddFakeCmd(&ovntest.ExpectedCmd{
+					Cmd:    "ovn-nbctl --timeout=15 --if-exists get logical_switch " + nodeName + " external_ids:gateway_ip",
+					Output: fmt.Sprintf("%q", nodeGWCIDR),
+				})
+				fexec.AddFakeCmd(&ovntest.ExpectedCmd{
+					Cmd:    "ovn-nbctl --timeout=15 get logical_switch_port " + portName + " dynamic_addresses",
+					Output: `"` + podMAC + " " + podIP + `"`,
+				})
+
+				err := util.SetExec(fexec)
+				Expect(err).NotTo(HaveOccurred())
+
+				_, err = config.InitConfig(ctx, fexec, nil)
+				Expect(err).NotTo(HaveOccurred())
+
+				fakeClient := fake.NewSimpleClientset(&v1.PodList{
+					Items: []v1.Pod{*newPod(namespace, podName, nodeName)},
+				})
+
+				err = util.SetExec(fexec)
```

Comment:
You do `InitConfig` + `SetExec` twice here (both before and after creating `fakeClient`). Cut and paste error or something? (Likewise in the other tests.)